utilities/test_suite/HIP/runTests.py

```diff
@@ -51,26 +51,22 @@ def get_log_file_list(preserveOutput):
 
 # Functionality group finder
 def func_group_finder(case_number):
-    if case_number < 5 or case_number == 13 or case_number == 36 or case_number == 45:
```

Comment:
Please remove all this in L53-L69. This is all you need:
```
def func_group_finder(case_number):
    for key, value in ImageAugmentationGroupMap.items():
        if case_number in value:
            return key
    return "miscellaneous"
```

Reply:
Done